src/View/Components/Input.php

```diff
@@ -42,20 +45,22 @@ public function __construct(
         ?string $suffix = null,
         ?string $prepend = null,
         ?string $append = null,
-        bool $errorless = false
+        bool $errorless = false,
+        bool $spinner = false
     ) {
         $this->borderless = $borderless;
         $this->shadowless = $shadowless;
-        $this->label      = $label;
-        $this->hint       = $hint;
+        $this->label = $label;
+        $this->hint = $hint;
         $this->cornerHint = $cornerHint;
-        $this->icon       = $icon;
-        $this->rightIcon  = $rightIcon;
-        $this->prefix     = $prefix;
-        $this->suffix     = $suffix;
-        $this->prepend    = $prepend;
-        $this->append     = $append;
-        $this->errorless  = $errorless;
+        $this->icon = $icon;
+        $this->rightIcon = $rightIcon;
+        $this->prefix = $prefix;
+        $this->suffix = $suffix;
+        $this->prepend = $prepend;
+        $this->append = $append;
+        $this->errorless = $errorless;
+        $this->spinner = $spinner;
```

Comment:
Please, use the phpcs fixer to format this file. Wireui is using it for code style.